package.json

```diff
@@ -56,11 +56,13 @@
     "onCommand:prettier-sql-vscode.format-selection"
   ],
   "main": "./out/extension.js",
+  "browser": "./out/extension.js",
```

Comment:
Do I understand correctly, that this is the only line that's actually needed to make this extension work in web?

The rest of the changes seem to be only about adding a commands to test it on web.

Reply:
Yes pretty much. I think.